packages/nx-plugin/src/generators/plugin/plugin.ts

```diff
@@ -163,5 +163,13 @@ export async function pluginGenerator(host: Tree, schema: Schema) {
   return runTasksInSerial(...tasks);
 }
 
+function resolveNpmScope(npmScope: string, name: string): string {
```

Comment:
This function resolves the package name, not the npmScope right?
```suggestion
function resolvePackageName(npmScope: string, name: string): string {
```

Reply:
yeah it's kinda in between npm scope / package name. But agree, fixing it 🙂